cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -525,3 +536,38 @@ func (c *CStorPoolController) getDeviceIDs(csp *apis.CStorPool) ([]string, error
 	// TODO: Add error handling
 	return pool.GetDeviceIDs(csp)
 }
+
+func (c *CStorPoolController) upgrade(csp *apis.CStorPool) (*apis.CStorPool, error) {
+	if csp.VersionDetails.Current != csp.VersionDetails.Desired &&
+		csp.VersionDetails.Desired != "" {
```

Comment:
where is I/P validation done?

Reply:
For I/P validation do we take RC builds into consideration ?

Reply:
yeah.. you can have

Reply:
Done